wp-content/themes/bpr2018/resources/assets/sass/_images.scss

```diff
@@ -18,6 +18,11 @@
   margin-left: $spacing-xs;
 }
 
+.lazy {
```

Comment:
It would be nice to give this class a more descriptive name like `lazy-image`, just so that this is clearer to future developers.